src/components/footer/footer-styles.js

```diff
@@ -0,0 +1,208 @@
+import styled from "styled-components"
+import { white, navyBlue, royalBlue } from "../../constants/colors"
+import footerHex from "../../images/footer-hex.svg"
+import { fonts, fontWeights } from "../../constants/typography"
+import { size } from "../../constants/devices"
+
+export const FooterContainer = styled.div`
+  background-color: ${navyBlue};
+  padding: 40px 100px 40px 100px;
+  font-family: "Work Sans", sans-serif;
+  color: white;
+  background-image: url("${footerHex}");
+  background-size: cover;
+  background-repeat: no-repeat;
+  @media only screen and (max-width: ${size.tablet}px) {
+    padding: 40px;
+  }
+`
+export const FooterRowContainer = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: stretch;
+`
+
+export const FooterLinks = styled.div`
+  color: white;
+  column-count: 2;
+  margin-top: auto;
+  @media only screen and (max-width: ${size.laptop}px) {
+    column-count: 1;
+  }
+`
+
+export const FooterLink = styled.a`
+  font-family: "Work Sans", sans-serif;
```

Comment:
^You can also abstract the font out into the `typography.js` file so we can reference the footer font variable here